tools/juliapkg/src/scalar_udf.jl

```diff
@@ -0,0 +1,55 @@
+# ctypes.jl
+const duckdb_scalar_function = Ptr{Cvoid}
+
+# api.jl scalar functions
+function duckdb_create_scalar_function()
+    return ccall((:duckdb_create_scalar_function, libduckdb), duckdb_scalar_function, ())
```

Comment:
Can we keep all the api functions in `api.jl` for consistency?

Reply:
Ofc!